jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/ddl/TestDBPlatformHelper.java

```diff
@@ -48,19 +46,49 @@ public void test() {
     @Test
     public void testDB2ZOS() {
         //Returned from jcc driver
-        Assert.assertEquals(DB2ZPlatform.class.getName(), getPlatformClass("DB2", "DSN10015"));
+        Assert.assertEquals(org.eclipse.persistence.platform.database.DB2ZPlatform.class.getName(), getPlatformClass("DB2", "10", "DSN10015"));
     }
 
     @Test
     public void testDB2I() {
         //Returned from jcc driver (DRDA)
-        Assert.assertEquals(DB2MainframePlatform.class.getName(), getPlatformClass("AS", "QSQ07020"));
+        Assert.assertEquals(org.eclipse.persistence.platform.database.DB2MainframePlatform.class.getName(), getPlatformClass("AS", "10", "QSQ07020"));
 
         //Returned from type 2 native driver & type 4 open source driver (non-DRDA)
-        Assert.assertEquals(DB2MainframePlatform.class.getName(), getPlatformClass("DB2 UDB for AS/400", "07.02.0000 V7R2m0"));
+        Assert.assertEquals(org.eclipse.persistence.platform.database.DB2MainframePlatform.class.getName(), getPlatformClass("DB2 UDB for AS/400", "07.02.0000", "V7R2m0"));
     }
 
-    private String getPlatformClass(String productName, String productVersion){
-        return DBPlatformHelper.getDBPlatform(productName + productVersion, log);
+    @Test
+    public void testMySQL() {
+        //Returned from jcc driver (DRDA)
+        Assert.assertEquals(org.eclipse.persistence.platform.database.MySQLPlatform.class.getName(), getPlatformClass("MySQL", "5", "5.5.5-10.1.33-MariaDB"));
```

Comment:
MariaDB is a fork off the MySQL project -- probably should test for both?

Reply:
I tested against the databases I have available to collect the metadata from. I don't have MariaDB available.

Reply:
For local testing I am using MariaDB 10.3 at Debian 10 (Buster) distribution.  At this version all LRG tests are OK. I think, that this version is compatible with MySQL 5.3./5.x I am not sure about higher versions.